src/main/java/com/portatlas/request/RequestMethod.java

```diff
@@ -0,0 +1,20 @@
+package com.portatlas.request;
+
+public enum RequestMethod {
+    GET("GET"),
```

Comment:
@tbaik Currently I need my request methods to be of type String. I guess in this case I could just make it a Constant instead of an enum.